lib/benchmarking/moab_to_catalog_testing.rb

```diff
@@ -26,13 +29,16 @@
   logger.info "at revision #{commit}"
 end
 sleep(10)
-# storage_dir = 'spec/fixtures/moab_storage_root'
-storage_dir = '/services-disk12/sdr2objects'
-time_to_check_existence = Benchmark.realtime do
-  RubyProf.start
-  logger.info "Check for output"
-  StorageRootToDB.check_moab_to_catalog_existence(storage_dir)
-  @prof = RubyProf.stop
+# storage_dir = 'spec/fixtures/storage_root01/moab_storage_trunk'
+# storage_dir = '/services-disk12/sdr2objects'
+Stanford::StorageServices.storage_roots.each do |storage_root|
```

Comment:
this is reading from whatever is in Settings and thus in config initializer, yes?  Might be good to indicate with a comment.

Reply:
will do!